sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -167,599 +153,6 @@ case class DescribeDatabaseCommand(
   }
 }
```

Comment:
ok~ thanks, another problem is now  the lines of tables.scala is 1594, this is too large, should we need to split it to reduce the  lines of a file?